screens/LoginScreen.js

```diff
@@ -1,10 +1,45 @@
-import { Text, View, StyleSheet, ImageBackground } from 'react-native';
+import { Text, View, StyleSheet, ImageBackground, Alert } from 'react-native';
 import Input from '../components/Auth/Input';
 import SecondaryButton from '../components/ui/SecondaryButton';
 import { useNavigation } from '@react-navigation/native';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  clearUserInputFields,
+  onBodyChangeEmail,
+  onBodyChangePassword,
+} from '../redux/userSlice';
+import { getUser, signInUser } from '../components/Auth/Services/client';
+import {
+  authenticateUser,
+  setAuthenticatedUser,
+  setUserAuthenticationToken,
+} from '../redux/authSlice';
 
 export default function LoginInScreen() {
   const navigation = useNavigation();
+  const dispatch = useDispatch();
+  const { email, password } = useSelector((state) => state.user);
+
+  async function submitHandler() {
+    const User = {
+      email: email,
+      password: password,
+    };
+    try {
+      const token = await signInUser(User);
+      dispatch(setAuthenticatedUser(await getUser()));
+      dispatch(setUserAuthenticationToken(token));
+      dispatch(authenticateUser(true));
```

Comment:
Would it make sense to condense any of these actions? For example, setting authenticated user and the token as part of the same action.